studyManager.py

```diff
@@ -15,17 +15,29 @@
 from include.calibrationLib import load_configuration
 
 
-# Get a list of all the studies from the database
+# Get list of all the studies from the database
 def get_studies(connectionString):
     sql = 'SELECT id, name FROM STUDY ORDER BY id ASC'
     return database.select(connectionString, sql, '')
 
 
 # Add the indicated study to the database
 def add_study(connectionString, studyName):
-    sql = 'INSERT INTO study(Name) VALUES(%s) RETURNING id;'
-    return database.insert_returning(connectionString, sql, {'Name': studyName})
-
+    sql = [('INSERT INTO study(Name) VALUES(%s) RETURNING id;', (studyName,))]
```

Comment:
This is much more complicated than it was before, which makes it much more likely that you are going to introduce errors into the code. Remember, don't make the code more complicated than it has to be.